learing_journal/templates/new-entry.jinja2

```diff
@@ -1,23 +1,23 @@
 {% extends "layout.jinja2" %}
 {% block body %}
 
-<div class="jumbotron">
-  <h1>new-entry.jinja2</h1>
+  <h4>new-entry.jinja2</h4>
   <!-- TODO: Wrap the back to home link in a button -->
   <h2><a href='/'>Back to Home</a></h2>
```

Comment:
`request.route_url` for this link. Also it's probably easier on you to just put the link in the header.